src/devices/Interop/Unix/Libc/Interop.close.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+internal partial class Interop
```

Comment:
Please move all of those to Interop.Libc.cs since this seems a bit too granular. Also for the common stuff please also move out stuff from https://github.com/dotnet/iot/blob/master/src/devices/SocketCan/Interop.cs with whatever is common (I can see: ioctl and close at minimum but you might want to review that)